### PR TITLE
Escape dots in URL regular expressions.

### LINKED ISSUE #132: Beware non-regex dots in default license normalizer config
Minor nitpick... I noticed that https://github.com/jk1/Gradle-License-Report/blob/c463064d013659273727f266eb6e5c04c0354a38/src/main/resources/default-license-normalizer-bundle.json#L45 contains an unescaped dot. Please correct this and review other lines for a similar error.

### PR DESCRIPTION
**Pull request by cowwoc**
Fixes #132 

Double-check line 60 in particular. I believe it's correct.

### CODE REVIEW

File: src/main/resources/default-license-normalizer-bundle.json [hunk 1]
```diff
@@ -57,9 +57,9 @@
     { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GNU General Public License, [v|V]ersion 2.*[c|C]lasspath [e|E]xception" },
     { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GNU General Public License, [v|V]ersion 2.*[c|C][p|P]?[e|E]" },
     { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GNU General Public License, [v|V]ersion 2.*, with the classpath exception" },
-    { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GPL2 w\/ CPE" },
-    { "bundleName" : "lgpl2.1",     "licenseUrlPattern" : ".*www.opensource.org/licenses/LGPL-2\\.1" },
-    { "bundleName" : "lgpl2.1",     "licenseUrlPattern" : ".*www.gnu.org/licenses/lgpl-2\\.1.html" },
+    { "bundleName" : "gpl2+ce",     "licenseNamePattern" : "GPL2 w/ CPE" },
```
Comment (guenhter) [inline src/main/resources/default-license-normalizer-bundle.json]: Yep, fine this way

File: src/main/resources/default-license-normalizer-bundle.json [hunk 2]
```diff
@@ -57,9 +57,9 @@
...
+    { "bundleName" : "lgpl2.1",     "licenseUrlPattern" : ".*www\\.opensource\\.org/licenses/LGPL-2\\.1" },
+    { "bundleName" : "lgpl2.1",     "licenseUrlPattern" : ".*www\\.gnu\\.org/licenses/lgpl-2\\.1.html" },
```
Comment (guenhter) [inline src/main/resources/default-license-normalizer-bundle.json]: Is there still one dot unquoted at the end?

Comment (cowwoc) [reply src/main/resources/default-license-normalizer-bundle.json]: Good catch. Fixed.

**Comment by jk1** [comment]
Thanks. Please find it merged.

**Pull request merged**